lib/kubeclient/common.rb

```diff
@@ -229,6 +229,31 @@ def retrieve_all_entities(entity_types)
       end
     end
 
+    def get_pod_log(pod_name, namespace, container: nil, previous: false)
+      params = {}
+      params[:previous] = true if previous
+      params[:container] = container if container
+
+      ns = build_namespace_prefix(namespace)
+      handle_exception do
+        rest_client[ns + "pods/#{pod_name}/log"]
+          .get({ 'params' => params }.merge(@headers))
+      end
+    end
+
+    def watch_pod_log(pod_name, namespace, container: nil)
+      params = { follow: true }
```

Comment:
@jonmoter is this a kubernetes api issue? Does `watch` always require `follow`? What if you're not passing it? Does it even make sense?